src/util/numeric.h

```diff
@@ -149,6 +149,16 @@ inline void sortBoxVerticies(v3s16 &p1, v3s16 &p2) {
 		SWAP(s16, p1.Z, p2.Z);
 }
 
+inline void sortBoxVerticies(v3f &p1, v3f &p2)
```

Comment:
Needs to have a differernt name from the function above to be clear it is for floats.
`sortBoxFloatVertices()`
Both functions have 2 arguments so can be easily confused.